### PR TITLE
Remove methods that do two things

### PR DESCRIPTION
**Pull request by mernst**
Also move getClasses from ASceneWrapper to AScene

Merge together with https://github.com/kelloggm/checker-framework/pull/242

### CODE REVIEW

File: scene-lib/src/scenelib/annotations/el/AScene.java [hunk 1]
```diff
@@ -141,6 +142,16 @@ public int hashCode() {
         return classes.hashCode() + packages.hashCode();
     }
 
+    /**
+     * Fetch the classes in this scene, represented as AClass objects.
+     *
+     * @return an immutable map from binary names to AClass objects
+     */
+    @SuppressWarnings("signature:return.type.incompatible") // unannotated ImmutableMap??
```
Comment (kelloggm) [inline scene-lib/src/scenelib/annotations/el/AScene.java]: the `classes` field isn't annotated, but should be.

Comment (kelloggm) [reply scene-lib/src/scenelib/annotations/el/AScene.java]: We could remove this warning suppression if we annotated it, but that would introduce more work (and annotations) elsewhere. I'm okay with leaving this suppression if you don't want to make more changes in scene-lib, but we should at least update this comment to say why the warning is needed (`classes` is unannotated).

**Review by kelloggm — CHANGES_REQUESTED** [review]
My only concern is the suppressed signature warning.

File: scene-lib/src/scenelib/annotations/el/AScene.java (hunk 1 above)
Comment (mernst) [reply scene-lib/src/scenelib/annotations/el/AScene.java]: scene-lib is not verified, so I should not have copied this warning suppression from the Checker Framework.

**Comment by mernst** [comment]
@kelloggm You can take another look when you get a chance.

**Review by kelloggm — APPROVED** [review]

**Pull request merged**